client/Stores/EventTypeAuthorization.elm

```diff
@@ -88,7 +88,7 @@ collectionDecoder : Decoder Authorization
 collectionDecoder =
     decode Authorization
         |> required "readers" (list (memberDecoder readPermission))
-        |> required "writers" (list (memberDecoder writePermission))
+        |> optional "writers" (list (memberDecoder writePermission)) []
```

Comment:
I think that `writers` is still required for event type authorization, isn't it?

Reply:
It is now just general "Authorization". Probably it is better to rename it.

Reply:
Thanks for the changes. Does this mean that it will now be possible to try and create/update an event type with empty writers? Nakadi will reject that

Reply:
it was always possible, the validation happens on the nakadi api side. This decoder used only for reading auth data form nakadi( parsing json into the internal structure). The structure is the same for both event type and subscription, but "writers" is always an empty array and ignored.